src/Compilers/Core/Portable/DiagnosticAnalyzer/AnalysisState.cs

```diff
@@ -241,8 +246,7 @@ private void UpdateEventsMap_NoLock(CompilationEvent compilationEvent, bool add)
 
         private void AddPendingSourceEvent_NoLock(SyntaxTree tree, CompilationEvent compilationEvent)
         {
-            HashSet<CompilationEvent> currentEvents;
-            if (!_pendingSourceEvents.TryGetValue(tree, out currentEvents))
```

Comment:
📝 The only necessary change here was adding a `?` to the `currentEvents` declaration. Inlining it caused unnecessary noise for the pull request.

Reply:
Same reason as above. Lot of these would have required changing the variable type to add an explicit `?` at the end of the type declaration syntax above. Inlining it makes it much cleaner.